.github/workflows/docker-tests.yml

```diff
@@ -5,27 +5,45 @@ on:
       - main
     paths-ignore:
       - '**/*.md'
-  pull_request:
+  pull_request_target:
```

Comment:
I thought this is no longer needed with the `${GITHUB_REF:-test}` approach?

Reply:
I tried that approach and there are some issues. this is temporary. will handle this in separate PR. need to work around it a bit.